package-list-brew

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
+echo "Setting things to install Bassa..."
 
 echo "Installing Homebrew"
-/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+/usr/bin/ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```

Comment:
This is broken due to the extra space before `ruby` in the url

Reply:
@demon-36 
Can you please fix that and **good practice would be to test it before sending a PR** 😀

Reply:
@kmehant after going through ruby scripts syantax I think you are correct bro, earlier I thought this script would install Homebrew in mentioned directory(`/usr/bin`) using `ruby` as command parameter,